### ISSUE TITLE
Problem with scrollpane extension

### ISSUE THREAD

**Issue by alex920** [edvin/tornadofx#147]
Looks like there are a downcasting problem with ScrollPane class:
```kotlin
root.scrollpane(TilePane()) {
            // some code
        }
```

At runtime jvm throws exception:
`java.lang.ClassCastException: javafx.scene.control.ScrollPane cannot be cast to javafx.scene.layout.TilePane`

**Comment by edvin** [comment]
Thanks for reporting this. I can confirm that I see the same thing and I'm working on a solution right now.

**Comment by edvin** [comment]
This issue has made us realise that we should make the whole builder structure more robust. We have a plan for it now, but in the mean time you should avoid using a builder for scrollpane and instantiate it manually, like this:

```kotlin
root += ScrollPane().apply {
    content = TilePane().apply {
        // Do something with TilePane here
    }
}
```

**Comment by edvin** [comment]
The new syntax for this will be a bit cleaner:

```kotlin
scrollpane {
    tilepane {
        // Add something to the TilePane here
    }
}
```

**Comment by edvin** [comment]
The first iteration of the reorganisation is complete. If you like, you can check out this branch to test it in your project:

**Comment by edvin** [comment]
You can also check this out directly if you're using Gradle, just include this in your `build.gradle`:

```groovy
repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile 'com.github.edvin:tornadofx:769d6f0b4246a0a72fd86dc3739bf5db6bcb2c96'
}
```

The dependency is instead of the normal TornadoFX dependency. We're still testing this feature but everything looks good, so hopefully we'll do a release within a few days.

**Comment by edvin** [comment]
The fix as described above is fixed in 1.5.3-SNAPSHOT, available in the master branch now :)
